packages/block-library/src/button/style.scss

```diff
@@ -27,7 +27,6 @@ $blocks-button__height: 56px;
 	padding: 12px 24px;
 	text-align: center;
 	text-decoration: none;
-	white-space: normal;
```

Comment:
For posterity's sake: I investigated why this was here in the first place. It was introduced in #5662. I verified the change here does not regress #5628.

![image](https://user-images.githubusercontent.com/1779930/56044670-b7231b00-5d0d-11e9-9b9e-772e6ca2d840.png)

Reply:
Thank you!